src/Raven.Client/Documents/Session/InMemoryDocumentSessionOperations.cs

```diff
@@ -197,6 +197,8 @@ public async Task<ServerNode> GetCurrentSessionNode()
             new Dictionary<(string, CommandType, string), ICommandData>();
 
         public readonly bool NoTracking;
+       
+        public Dictionary<string, ForceRevisionStrategy> IDsForCreatingForcedRevisions = new Dictionary<string, ForceRevisionStrategy>(StringComparer.OrdinalIgnoreCase);
```

Comment:
can be private or internal?

Also rename to 'IdsForCreatingForcedRevisions'

Reply:
Done. Renamed and made internal.